routetable/route_table.go

```diff
@@ -347,6 +355,45 @@ func (r *RouteTable) syncRoutesForLink(ifaceName string) error {
 	return nil
 }
 
+// startConntrackDeletion starts the deletion of conntrack entries for the given CIDR in the background.  Pending
+// deletions are tracked in the pendingConntrackCleanups map so we can block waiting for them later.
```

Comment:
I think we need a bit more context somewhere to record _why_ we're making this change to move conntrack deletions into the background.  I'd be happy with any of:
- adding to the comment here
- adding to the commit message
- adding more to the PR comments - either inline explanation or a link to an issue where it's explained.

At the moment - unless I missed it - I don't think we have any of those.